src/replicator.js

```diff
@@ -10,15 +10,15 @@ import {
   Account,
   SystemProgram,
 } from '@solana/web3.js';
-import fs from 'mz/fs';
-import {solanaInstallInit} from './solana-install-init';
-import {sleep} from './sleep';
-import {url} from './url';
+import fs from 'fs';
```

Comment:
Why was `mz/fs` changed to `fs`?  There's an `await fs.access()` in this file so that's not going to work with `fs` right?